twindle-cli/twitter-puppeteer/index.js

```diff
@@ -1,19 +1,59 @@
-const puppeteer = require('puppeteer');
-const getTweet = async (pageUrl) => {
-  const browser = await puppeteer.launch({ headless: true });
-  const page = await browser.newPage();
-  await page.goto(pageUrl, {
-    waitUntil: 'networkidle0',
-  });
-  const tweets = await page.evaluate(() => {
-    const tweetNodes = Array.from(
-      document.querySelectorAll('div[lang]:not([lang=""])')
-    );
-    const data = tweetNodes.map((node) => ({
-      tweet: node.innerText,
-    }));
-    return data;
-  });
-  return { data: tweets };
+const puppeteer = require("puppeteer");
+const { waitFor } = require("../utils/helpers");
+
+/**
+ * Get tweets(even older than 7 days) using puppeteer
+ * @param {string} tweetID
+ */
+const getTweet = async (tweetID) => {
+  const pageURL = `https://twitter.com/anyone/status/${tweetID}`;
```

Comment:
Does this enable accessing any tweet without knowing the user?

Reply:
Yup. Twitter will automatically redirect to the right person. You could replace anyone with realDonaldTrump 🤣🤣🤣

(DON'T, BUT)